ONSdigital/design-system
# Remove border left on warning panels

### What is the context of this PR?
![image](https://user-images.githubusercontent.com/42928680/98663218-b4046c80-2340-11eb-8950-ffac2d67a0c6.png)

Noticed this issue with footer warning panels. This PR will make sure the unneeded margin left is removed. This will also make sure warning panels align with the edge of the page.

### How to review 
- Check footer warning looks as it should without the border left
- Check that warning panels no longer have a border left and align with edge of page
